backend/TraderX/src/main/java/cmpe451/group6/rest/follow/repository/FollowRepository.java

```diff
@@ -0,0 +1,28 @@
+package cmpe451.group6.rest.follow.repository;
+
+import cmpe451.group6.authorization.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import cmpe451.group6.rest.follow.model.FollowDAO;
+
+import java.util.List;
+
+public interface FollowRepository extends JpaRepository<FollowDAO, Integer> {
+
+    // List<FollowDAO> findByFollower(User follower); //it doesn't work, I don't
+    // know why.
+    // List<FollowDAO> findByFollowee(User followee); //it doesn't work, I don't
+    // know why.
```

Comment:
I think we have to make sure why it does work or not

Reply:
We have removed the unnecessary parts. I think this issue is resolved. Do you agree with me @barisegesevgili ?

Reply:
Seems like underscore is reserved character in JPArepository. This is the possible reason of this.